File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string res="";
        int i=0;
        while(i<word1.length() || i<word2.length()){
            if(i<word1.length()){
                res+=word1[i];
            }
            if(i<word2.length()){
                res+=word2[i];
            }
            i++;
        } 
        return res;
    }
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        ans=[]
        i, j = 0, 0
        while i<len(word1) and j<len(word2):
            ans.append(word1[i])
            ans.append(word2[j])
            i+=1
            j+=1
        ans.append(word1[i:])
        ans.append(word2[j:])
        return "".join(ans)